manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+never be ahead of Staging or Integration.
+
+### Step 1: Merge your branch
+
+This assumes that you have create a pull request for your changes in GitHub, and it has been approved by a reviewer. TDR
+repos are set up so that you can only merge a branch once it has been approved and the CI checks have succeeded (see the
+[repo setup guide]).
+
+### Step 2: Deploy to Integration
+
+Most repos, including the main codebases like the frontend and consignment API, are deployed to Integration
+automatically after the main branch build has finished. If you are unsure, check the Jenkinsfile in the repo.
+
+Some repos, like the [database migrations], have a Jenkins deployment job that needs to be started manually. Check the
+repo's Readme for instructions.
+
+Before moving on to the next step, wait for the deployment to finish. Some deployments, such as changes to Lambda code
+and database migrations, happen immediately. So if the Jenkins job succeeded, then the change has been deployed.
+
+[ECS] deployments, such the frontend, consignment API and auth server, take a few minutes longer while the load balancer
+switches over to the new container. To check the progress of a deployment, open the [Grafana ECS dashboard] and look at
+the number of tasks running in Integration. When a deployment starts, the number of tasks jumps up from 1 to 2. When it
+has finished, it falls back down to 1. For example, this graph shows a Keycloak deployment followed by a frontend
+deployment:
+
+![A graph showing the rise then fall in the number of instances of ECS containers during a deployment](./images/ecs-deployment-graph.png)
+
+Alternatively, you can open the ECS service in the AWS console and check the Events tab. You should see logs when ECS
+starts a new task and stops the old one.
+
+### Step 3: Check the deployment on Integration
+
+Before deploying to Staging and Production, make sure that you're happy with the change on Integration. For most
+deployments, this means waiting for the [end-to-end tests][e2e-tests-intg] to run. You should also try out the change
+yourself on Integration.
+
+You should check for any alerts posted to the da-tdr-releases Slack channel that might indicate problems with the
+deployment. You could also check relevant [Grafana] graphs or CloudWatch logs to look for anything that's different to
+normal.
+
+You should not need to run manual regression tests. If you feel like you need to do that to be confident in the change,
+consider adding more automated tests that would check those things.
+
+Some changes may need to run for some time before we're confident that they work OK. This might be the case for file
+check changes that could affect the performance, or which might cause intermittent errors. In this case, warn the rest
+of the team that they should not deploy this repo to staging and production yet. Try to keep the testing time short
```

Comment:
Capital "S" and "P": ... Staging and Production yet. ...

Reply:
I went through the whole doc and fixed the environment capitalisation.